src/state/trajectory/types.ts

```diff
@@ -92,3 +106,18 @@ export interface TimeUnits {
     magnitude: number;
     name: string;
 }
+
+export interface HealthCheckTimeout {
+    [requestId: string]: NodeJS.Timeout;
+}
+
+type CONVERSION_NO_SERVER = typeof CONVERSION_NO_SERVER;
+type CONVERSION_SERVER_LIVE = typeof CONVERSION_SERVER_LIVE;
+type CONVERSION_ACTIVE = typeof CONVERSION_ACTIVE;
+type CONVERSION_COMPLETE = typeof CONVERSION_COMPLETE;
```

Comment:
This is really strange to me. Could this and line 119 be reduced? 

`ConversionStatus = typeof CONVERSION_NO_SERVER | typeof CONVERSION_SERVER_LIVE | ...`

Reply:
This is a good observation, to be honest I just copied the way it was done in viewer status.

But your suggestion will improve both. @meganrm do you see any reason that would cause an issue?

I'm going to leave the initial pipe because it looks too weird to me without it, but I'll change it if I have too lol

```
export type ConversionStatus =
    | typeof CONVERSION_INACTIVE
    | typeof CONVERSION_NO_SERVER
    | typeof CONVERSION_SERVER_LIVE
    | typeof CONVERSION_ACTIVE
    | typeof CONVERSION_COMPLETE;
    
    export type ViewerStatus =
    | typeof VIEWER_LOADING
    | typeof VIEWER_ERROR
    | typeof VIEWER_EMPTY
    | typeof VIEWER_SUCCESS;